### ISSUE TITLE
[BUG] iOS only shows black view

### ISSUE THREAD

**Issue by mkoehne** [sarbagyastha/youtube_player_flutter#302]
**Describe the bug**
The iOS Player shows only a black view. The same code works on Android.

**To Reproduce**
- Create a new project
- Add youtube_player_iframe
- Follows iOS steps to configure Info.plist
- Start App

**Expected behavior**
Show the player without the black view.

**Screenshots**
![youtube_player_iframe](https://user-images.githubusercontent.com/15260431/87961113-8ce60e80-cab5-11ea-9c9a-5d62176611d7.png)

**Technical Details:**
 - Device: iPhone 11 Pro (Simulator and real device)
 - OS: iOS 13.5
 - Version 1.0.1

**Comment by Shakle** [comment]
Same here

**Comment by geriby23** [comment]
Willl there be a fix to it? Or on real device it works and is not working just on emulators

**Comment by phackwer** [comment]
Have any of you remembered to request permission to stream video from the internet?

https://flutter.dev/docs/cookbook/plugins/play-video#2-add-permissions-to-your-app

**Comment by phackwer** [comment]
Also, if you read the docs you will see this:

![image](https://user-images.githubusercontent.com/4871786/88425413-3f381180-cde7-11ea-8a9c-3f0334f30428.png)

**Comment by mkoehne** [comment]
@phackwer Yeah I have requested the permission, like I wrote in the steps To Reproduce.
Even with the requested permission it is not working on my real iOS device.

**Comment by Shakle** [comment]
Works in 1.1.0. Many thanks!
